src/Authenticator/SessionAuthenticator.php

```diff
@@ -86,6 +86,7 @@ public function persistIdentity(ServerRequestInterface $request, ResponseInterfa
         $sessionKey = $this->getConfig('sessionKey');
         $session = $request->getAttribute('session');
         if (!$session->check($sessionKey)) {
+            $session->read();
```

Comment:
Since the middleware now persists identity after calling $next() (aka controller code has already run) is this still necessary?

Reply:
It probably isn't. I can take it out.